<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceInformationClass</class>
 <widget class="QWidget" name="DeviceInformationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1854</width>
    <height>1399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceInformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#FFFFFF;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="506,700,505">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="292,158,158,109,300">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="32,210,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_27">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/res/deviceinfo/device.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>PingFang SC</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:18px;color:#333333;font-family:PingFang SC;</string>
              </property>
              <property name="text">
               <string>设备信息</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,1,1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>设备型号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelDeviceType">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>type</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>设备ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelID">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>id</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>软件系统</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelSoftSys">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>windows10</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>内存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelMemorySize">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>256G</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>磁盘使用情况</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelDiskUse">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>1G/1T</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:1px solid #F0F0F0;</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="32,78,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_26">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/res/deviceinfo/version.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="styleSheet">
               <string notr="true">font-size:18px;color:#333333;font-family:PingFang SC;</string>
              </property>
              <property name="text">
               <string>软/硬件版本信息</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>软件版本</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelSoftVersion">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>IPS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>硬件SN</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelHardSN">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>SN</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:1px solid #F0F0F0;</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="32,84,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/res/deviceinfo/producer.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="styleSheet">
               <string notr="true">font-size:18px;color:#333333;font-family:PingFang SC;</string>
              </property>
              <property name="text">
               <string>生产厂家</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="177,523">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>生产厂家</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelManufacturer">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>上海工物高技术产业发展有限公司</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="177,523">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="styleSheet">
                 <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
                </property>
                <property name="text">
                 <string>厂家地址</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelManufacturerAddr">
                <property name="styleSheet">
                 <string notr="true">color:#333333;font:14px PingFang SC;</string>
                </property>
                <property name="text">
                 <string>中国(上海)自由贸易试验区......</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:1px solid #F0F0F0;</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="26,28">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/res/deviceinfo/address.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="styleSheet">
               <string notr="true">font-size:18px;color:#333333;font-family:PingFang SC;</string>
              </property>
              <property name="text">
               <string>使用地址</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="177,523">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="styleSheet">
               <string notr="true">color:#999999;font:14px PingFang SC;margin-left:43px;</string>
              </property>
              <property name="text">
               <string>使用地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelDispositionAddr">
              <property name="styleSheet">
               <string notr="true">color:#333333;font:14px PingFang SC;</string>
              </property>
              <property name="text">
               <string>上海浦东国际机场</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
